lib/server/backends/redis.ts

```diff
@@ -126,7 +127,7 @@ export async function getOrCreateSession(redis: Redis, req: ValidatedRequest): P
   const sessionsSetKey = makeKey([siteName, "sessions"]);
   let session: SessionDescriptor | null = null;
 
-  if (exists(sid)) {
+  if (exists(sid) && sid !== "") {
```

Comment:
```suggestion
  if (exists(sid) && sid !== "" && sid !== `""`) {
```

Reply:
Seems we still need to catch if the sid returns empty quotes as a string. I'm seeing a bunch of `[IB] Invalid or unknown session 'demo:session:""'`

Reply:
Thanks @tbakerx , It's just from the "clear and reload" button in the landing page. I've made the session cookie HttpOnly now, and I yanked that button. You'll have to use DevTools to clear your session now.